Update CODE_OF_CONDUCT.md (#1704)
hackclub/hackclub: CODE_OF_CONDUCT.md

  - discrimination based on age, nationality, race, (dis)ability, gender (identity or expression), sexuality, religion, or similar personal characteristic;
  - bullying or systematic harassment;
  - unwelcome sexual advances, including sexually explicit content.

> - Advertisement for events, companies, organizations, etc- unless specifically given permission by Staff, Community Team, or a Hack Club Community Game Designer.

Reviewer: ```suggestion
- Advertisement for events, companies, organizations, etc- unless specifically given permission by Hack Club HQ.
```
